popde/utils_plot.py

```diff
@@ -0,0 +1,164 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+
+def simple2Dplot(xx, yy, kde):
+    """
+    return kde plot in 2D given output
+    """
+    fig = plt.figure()
+    ax = fig.gca()
+    # Contourf plot
+    cfset = ax.contourf(xx, yy, kdevals, cmap='Blues')
+    # Contour plot
+    cset = ax.contour(xx, yy, kdevals, colors='k')
+    # Label plot
+    ax.clabel(cset, inline=1, fontsize=10)
+    ax.set_xlabel('Y1')
+    ax.set_ylabel('Y0')
+
+    plt.show()
+
+
+class KDEPlotter:
```

Comment:
Some of the functions here could be useful, but I don't see why this is a class. The point of making a class is to organize information that belongs together and should persist for some time. The density estimate class already does this for the data points and the KDE.

Reply:
removed